kansha/board/boardconfig.py

```diff
@@ -63,9 +65,10 @@ def __init__(self, board):
         """
         self.board = board
         self.labels = []
-        for data in board.labels:
-            t = component.Component(label.LabelTitle(data))
-            l = component.Component(data, model='edit-color')
+        for lbl in board.labels:
+            t = component.Component(title.EditableTitle(lbl.get_title))
+            t.on_answer(lbl.set_title)
```

Comment:
> Le 3 déc. 2015 à 14:00, Romuald Texier-Marcadé <notifications@github.com> a écrit :
> 
> In kansha/board/boardconfig.py:
> 
> > @@ -63,9 +65,10 @@ def __init__(self, board):
> >          """
> >          self.board = board
> >          self.labels = []
> > -        for data in board.labels:
> > -            t = component.Component(label.LabelTitle(data))
> > -            l = component.Component(data, model='edit-color')
> > +        for lbl in board.labels:
> > +            t = component.Component(title.EditableTitle(lbl.get_title))
> > +            t.on_answer(lbl.set_title)
> 
> Don't we apply any limit on line width?

We do. But not 72 like in the « old ages » of the first version of the PEP8.

We follow the latest version, with a max length of 99:

"
Some teams strongly prefer a longer line length. For code maintained exclusively
or primarily by a team that can reach agreement on this issue, it is okay to
increase the nominal line length from 80 to 100 characters (effectively increasing
the maximum length to 99 characters), provided that comments and docstrings are
still wrapped at 72 characters.
"

(https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

> 
> —
> Reply to this email directly or view it on GitHub.
>